mopro-ffi/src/app_config/utils.rs

```diff
@@ -0,0 +1,67 @@
+use std::path::{Path, PathBuf};
+use std::process::Command;
+use std::{fs, io};
+use uuid::Uuid;
+
+pub fn build_release() -> io::Result<()> {
```

Comment:
This name is very ambiguous, we're building like 10 different things in this module. `build_release` in the `utils` file tells me very little.

Reply:
Would `build_cdylib` better represent it?

Reply:
Sure, that name is good in the `mod.rs` file

Reply:
Ended up putting everything back in place after re-introducing platform-specific functions. It is better the original way